wsi_superpixel_guided_labeling/web_client/views/vue/components/store.js

```diff
@@ -73,10 +78,14 @@ const updatePixelmapLayerStyle = (overlayLayers) => {
                     return 'rgba(0, 0, 0, 0)';
                 }
                 const category = store.categories[d];
-                // If opacity is zero, fill color and stroke color should be the same
-                let strokeColor = `rgba(${[0, 0, 0]}, ${store.strokeOpacity})`;
-                strokeColor = store.strokeOpacity ? strokeColor : category.fillColor;
-                return (i % 2 === 0) ? category.fillColor : strokeColor;
+                // Fade between black border and fill color when opacity is changed
+                const strokeColor = rgbStringToArray(category.strokeColor);
+                const rgba = _.map(rgbStringToArray(category.fillColor), (val, idx) => {
+                    // rgb values are missing alpha, default to opaque (1)
+                    const strokeValue = strokeColor[idx] || 1;
```

Comment:
Shouldn't this be `strokeColor[idx] === undefined ? 1 : strokeColor[idx]`, since otherwise 0 values get shifted to 1.

Reply:
Yes, fixed, thank you!